ios/FluentUI.Demo/FluentUI.Demo/DemoTableViewController.swift

```diff
@@ -22,7 +22,7 @@ class DemoTableViewController: UITableViewController {
     override func viewDidLoad() {
         super.viewDidLoad()
 
-        tableView.backgroundColor = TableViewCell.tableBackgroundGroupedColor
+        tableView.backgroundColor = TableViewCell.tableBackgroundGroupedColor(fluentTheme: view.fluentTheme)
```

Comment:
do we really want this? it makes the objc caller scenario worse that it was before